next.config.js

```diff
@@ -48,19 +48,13 @@ const securityHeaders = [
   },
 ]
 
-// GraphQL end point
-const graphQLurl = process.env.NEXT_CONTENT_GRAPHQL
-  ? process.env.NEXT_CONTENT_GRAPHQL
-  : ''
-
 module.exports = {
   images: {
     dangerouslyAllowSVG: true,
   },
 
   env: {
     NEXT_PUBLIC_BUILD_DATE: builddate,
-    NEXT_CONTENT_GRAPHQL: graphQLurl,
```

Comment:
Should you add the two new env variables here? Just a question

Reply:
Nah, if you set them up as NEXT_PUBLIC it does the same thing